index.src.html

```diff
@@ -1403,7 +1403,7 @@ <h3 id="network-leakage">Network Leakage</h3>
   one network to be sent while the user is on another network, even if they
   don't explicitly open the page from which the report was sent.
 
-  ISSUE(WICG/BackgroundSync#107): Consider mitigations. For example, we could
+  ISSUE(W3C/BackgroundSync#107): Consider mitigations. For example, we could
```

Comment:
GitHub urls are more-or-less case-insensitive, but `w3c` is the canonical location.